org-brain.el

```diff
@@ -2334,6 +2334,12 @@ If run interactively, toggle following on/off."
 (defvar-local org-brain--visualize-header-end-pos 0
   "Buffer position at end of headers (history etc) in `org-brain-visualize'.")
 
+(defun check-org-brain-exist-p ()
```

Comment:
Since there are no namespaces in Emacs, all symbols in a package should start with the package name, followed by a dash. So in this case the function should be name `org-brain-visualize-buffer-exists-p` or similar.

However I don't understand the implementation of this function, nor why it is needed. If you want to check if the `*org-brain*` buffer exists, you could do `(get-buffer "*org-brain*")` which will return the buffer, or `nil` if it doesn't exist.

Reply:
> However I don't understand the implementation of this function, nor
> why it is needed

The idea is the behavior of calling **Info** mode with <kbd>C-h i</kbd>.

When there is no **Info** buffer, just open a new **Info** buffer. However,
when there is an **Info** buffer, just switch to it. The content is what
was left.

I would like to have a similar feature like **Info**, because I
constantly switch in between **org-brain** buffer and my working
document. If this feature is in, I could use exactly the same
key-bind to do the same thing to open **org-brain**.

-   The function name has been changed in
    b1508e7f5c9a827a5623edfe0f0a4d417f7059b6.

-   Thanks for your help for checking **org-brain** buffer, I revise it in
    f6f1b9dabb284f13819f63d7158c08ba74f36631.